projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -804,13 +805,22 @@ export class IgxSliderComponent implements
             return;
         }
 
+        this._sliding = true;
         const activeThumb = this.thumbTo.isActive ? this.thumbTo : this.thumbFrom;
         activeThumb.nativeElement.setPointerCapture($event.pointerId);
         this.showSliderIndicators();
 
         $event.preventDefault();
     }
 
+    /**
+     * @hidden
+     */
+    public onPointerMove($event: PointerEvent) {
```

Comment:
This shouldn't be needed for much else, so either `private` or `protected` if used in template
```suggestion
    private onPointerMove($event: PointerEvent) {
```